deepchem/feat/graph_features.py

```diff
@@ -268,16 +274,20 @@ def pair_features(mol, edge_list, canon_adj_list, bt_len=6,
 
   Parameters
   ----------
-  mol: TODO
-    TODO
+  mol: RDKit Mol
+    Molecule to compute features on.
   edge_list: list
-    List of edges t oconsider
+    List of edges to consider
   canon_adj_list: list
     TODO
   bt_len: int, optional
     TODO
   graph_distance: bool, optional
-    TODO
+    If true, use graph distance between molecules. Else use euclidean distance.
+
```

Comment:
I think this function is important for graph model's featurizer.
it is better to add returns doc string.

Reply:
Added in descriptions!